src/System.Management.Automation/resources/Modules.resx

```diff
@@ -627,4 +627,49 @@
   <data name="CannotFindCoreCompatibleBuiltInModule" xml:space="preserve">
     <value>Cannot find the built-in module '{0}' that is compatible with the 'Core' edition. Please make sure the PowerShell built-in modules are available. They usually come with the PowerShell package under the $PSHOME module path, and are required for PowerShell to function properly.</value>
   </data>
+  <data name="WDACExportModuleCommandLogTitle" xml:space="preserve">
+    <value>Export-ModuleMember Cmdlet</value>
+  </data>
+  <data name="WDACExportModuleCommandLogMessage" xml:space="preserve">
+    <value>Export of module members will fail because module '{0}', has a different language mode '{1}' than the current session '{2}'.</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogTitle" xml:space="preserve">
+    <value>Module Implicit Function Export</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogMessage" xml:space="preserve">
+    <value>Implicit function export for module '{0}' will be denied, because it is trusted but the session is in ConstrainedLanguage mode (not trusted).</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogMessage2" xml:space="preserve">
+    <value>Implict function export for trusted module '{0}' will be denied because the session is in ConstrainedLanguage mode.</value>
+  </data>
+  <data name="WDACScriptFileImportLogTitle" xml:space="preserve">
+    <value>Importing Script File as Module</value>
+  </data>
+  <data name="WDACScriptFileImportLogMessage" xml:space="preserve">
+    <value>Importing the script file '{0}' as a module will be disallowed in ConstrainedLanguage mode.</value>
+  </data>
+  <data name="WDACModuleDotSourceLogTitle" xml:space="preserve">
+    <value>Module Contains Dot-Source Operator</value>
+  </data>
+  <data name="WDACModuleDotSourceLogMessage" xml:space="preserve">
+    <value>Module '{0}' import will fail because it exports functions using wild card characters while also uses the dot-source operator.</value>
```

Comment:
Not sure the intent of the message here. Option 1:
```suggestion
    <value>Module '{0}' import will fail because it exports functions using wildcard characters while also using the dot-source operator.</value>
```

Option 2:
```suggestion
    <value>Module '{0}' import will fail because it exports functions using wildcard characters and uses the dot-source operator.</value>
```